vsintegration/src/FSharp.Editor/Common/CommonHelpers.fs

```diff
@@ -41,6 +41,18 @@ type internal SymbolLookupKind =
     | ByRightColumn
     | ByLongIdent
 
+[<RequireQualifiedAccess>]
+module Option =
+    /// Gets the value associated with the option or the supplied default value.
+    let inline getOrElse v = function
```

Comment:
![pasted image at 2016_12_20 20_48](https://cloud.githubusercontent.com/assets/680051/21408184/9083d93e-c7a1-11e6-8e47-f8fa0c6d887d.png)

I've gotten a bunch of these too